src/Core/Config/Model/DbaUtils.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Friendica\Core\Config\Model;
+
+class DbaUtils
```

Comment:
Why not use the `Util` namespace?
```suggestion
namespace Friendica\Core\Config\Util;

class ValueConversion
```